packages/lshw.sh

```diff
@@ -0,0 +1,50 @@
+LSHW_VERSION="B.02.20"
+LSHW_CHECKSUM="5805eba5f31886582fff673c5dccdb3b"
+LSHW_LINK="https://github.com/lyonel/lshw/archive/refs/tags/${LSHW_VERSION}.tar.gz"
+
+download_lshw() {
+    download_file $LSHW_LINK $LSHW_CHECKSUM
```

Comment:
You need to use an extra argument specifying a custom archive filename (which is used as filename requested on the mirror server), to see to which package it belongs (the program name is not on the filename).

Because on the Github Release the name is only the version, the filename is `B.02.20.tar.gz`

Fix with:
```sh
download_file $LSHW_LINK $LSHW_CHECKSUM lshw-${LSHW_VERSION}.tar.gz
```